src/main/java/dev/aziz/movies/controllers/MovieController.java

```diff
@@ -20,11 +27,31 @@ public class MovieController {
 
     @GetMapping("/movies")
     public ResponseEntity<List<ReducedMovieDto>> getAllMovies() {
-        return new ResponseEntity<>(movieService.getMovies(), HttpStatus.OK);
+        return ResponseEntity.ok(movieService.getMovies());
     }
 
     @GetMapping("/movies/{id}")
     public ResponseEntity<MovieDto> getMovie(@PathVariable Long id) {
-        return new ResponseEntity<>(movieService.findMovieById(id), HttpStatus.OK);
+        return ResponseEntity.ok(movieService.findMovieById(id));
+    }
+
+    @PostMapping("/movies")
+    public ResponseEntity<MovieDto> createMovie(@RequestBody @Valid MovieDto movieDto) {
+        return ResponseEntity.created(URI.create("/movies")).body(movieService.createMovie(movieDto));
```

Comment:
The URL where you can find the created movie is not "/movies" but "/movies/{id}"